src/generate.js

```diff
@@ -84,11 +85,18 @@ self.generate = function(options) {
 	if (options.symbols) {
 		pool += symbols;
 	}
+
 	// similar characters
 	if (options.excludeSimilarCharacters) {
 		pool = pool.replace(similarCharacters, '');
 	}
 
+	// excludes characters from the pool
+	var i = options.excludes.length;
```

Comment:
Really down to personal preference. Either or works. Reverse while loops are meant to be more optimised.